index.html

```diff
@@ -3198,6 +3198,25 @@ <h3>Midpoint </h3><br>
                 class="stopwrap"
                align="right"></p>
             <h4 id="mid_output"> </h4> 
+            <h3>Distance between point and a line</h3><br>
+            <div class="coll">
+                <p>
+                    Line Equation :   <input type="text" id="a" size=3 value='' placeholder="A">  x  +
+                <input type="text" id="b" size=3 value='' placeholder="B"> y + <input type="text" id="c" size=3 value='' placeholder="C"> = 0
+                    <br>
```

Comment:
Can you add another `<br>` tag here so that the "Line Equation" and "Point" paragraphs do not look cluttered?

Reply:
Yes I'll add it soon.